arviz/tests/test_utils.py

```diff
@@ -239,3 +248,13 @@ def test_stack():
     assert x.shape[1:] == y.shape[1:]
     assert np.allclose(np.vstack((x, y)), _stack(x, y))
     assert _stack
+
+
+@pytest.mark.parametrize("data", [np.random.randn(1000), np.random.randn(1000).tolist()])
+def test_two_de(data):
+    assert np.allclose(two_de(data), np.atleast_2d(data))
+
+
+@pytest.mark.parametrize("data", [np.random.randn(100), np.random.randn(100).tolist()])
```

Comment:
What does the tolist check here? Can docstring be added? Same with the test above?

Reply:
tolist() is used to create a non ndarray object, to test the else condition.

Reply:
Good thinking